debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -71,9 +72,9 @@
     private static final String GET_ALL_CHANGES_FOR_TABLE = "SELECT *# FROM [#db].cdc.[fn_cdc_get_all_changes_#](?, ?, N'all update old') order by [__$start_lsn] ASC, [__$seqval] ASC, [__$operation] ASC";
     private final String get_all_changes_for_table;
     protected static final String LSN_TIMESTAMP_SELECT_STATEMENT = "TODATETIMEOFFSET([#db].sys.fn_cdc_map_lsn_to_time([__$start_lsn]), DATEPART(TZOFFSET, SYSDATETIMEOFFSET()))";
-    private static final String GET_LIST_OF_CDC_ENABLED_TABLES = "EXEC [#db].sys.sp_cdc_help_change_data_capture";
+    private static final String GET_LIST_OF_CDC_ENABLED_COLUMNS = "SELECT object_id, column_id, column_name FROM [#db].cdc.captured_columns ORDER BY object_id ASC, column_id ASC";
+    private static final String GET_LIST_OF_CDC_ENABLED_TABLES = "WITH EligibleCaptureInstances AS (SELECT ROW_NUMBER() OVER (PARTITION BY ct.source_object_id, ct.start_lsn ORDER BY ct.create_date DESC) AS CISequence, ct.* FROM [#db].cdc.change_tables AS ct#) SELECT OBJECT_SCHEMA_NAME(source_object_id, DB_ID(?)) AS source_schema, OBJECT_NAME(source_object_id, DB_ID(?)) AS source_table, capture_instance, object_id, start_lsn FROM EligibleCaptureInstances WHERE CISequence = 1";
```

Comment:
Can you

* wrap those queries so that they are a bit easier to read
* add a JavaDoc comment to each describing its purpose, in particular the deduplication logic is worth pointing out
* add a JavaDoc comment stating why we don't use the ready-made procedure; i.e. some of the (really great) info from this PR description should go into a code comment in a condensed form